src/components/ListView/ListViewRow.js

```diff
@@ -23,7 +23,7 @@ const ListViewRow = ({
   heading,
   description
 }) => (
-  <div style={{ display: 'flex', flex: 1 }}>
+  <div style={{ flex: 1 }}>
```

Comment:
Is it possible to remove this `<div>` altogether now that we have upgraded to React 16? I think the reason this solution works is because we have float fallbacks for browsers that don't support flex, so now removing `display: 'flex'` results in using those float fallbacks. But ultimately my preference would be to match the patternfly core html structure whenever possible.

Reply:
Agreed and updated 👍